userguide/directoryservices.rst

```diff
@@ -106,31 +106,27 @@ advanced options.
    +--------------------------+---------------+----------+-------------------------------------------------------------------------------------------------------------------------------+
    | Domain Account Password  | string        |          | Password for the Active Directory administrator account. Required the first time a domain is configured. After initial        |
    |                          |               |          | configuration, the password is not needed to edit, start, or stop the service.                                                |
-   |                          |               |          |                                                                                                                               |
    +--------------------------+---------------+----------+-------------------------------------------------------------------------------------------------------------------------------+
    | Encryption Mode          | drop-down     | ✓        | Choices are *Off*, *SSL (LDAPS protocol port 636)*, or *TLS (LDAP protocol port 389)*. See                                    |
    |                          |               |          | http://info.ssl.com/article.aspx?id=10241 and https://hpbn.co/transport-layer-security-tls/ for more information about SSL    |
    |                          |               |          | and TLS.                                                                                                                      |
    +--------------------------+---------------+----------+-------------------------------------------------------------------------------------------------------------------------------+
-   | Certificate              | drop-down     | ✓        | Select the Active Directory server certificate if SSL connections are used. If a certificate does not exist, create a         |
-   |                          | menu          |          | :ref:`Certificate Authority <CAs>`, then create a certificate on the Active Directory server. Import the certificate to the   |
-   |                          |               |          | %brand% system using the :ref:`Certificates` menu.                                                                            |
+   | Certificate              | drop-down     | ✓        | Select the Active Directory server certificate if SSL connections are used. If a certificate does not exist, create           |
+   |                          | menu          |          | or import a :ref:`Certificate Authority <CAs>`, then create a certificate on the Active Directory server. Import              |
+   |                          |               |          | the certificate to the %brand% system using the :ref:`Certificates` menu. This drop-down will most                            |
```

Comment:
"This drop-down will most likely not be used to configure LDAPs." reads a little awkward. How about reworking a little to include a positive recommendation instead of a dead end? "It is recommended to use {x} to configure LDAP instead of using this option."